src/Collection/Filter.php

```diff
@@ -0,0 +1,12 @@
+<?php namespace Lean\Endpoints\Collection;
+
+/**
+ * Custom filters used on this endpoint.
+ *
+ * @package Endpoints\Collection;
+ * @since 0.1.0
+ */
+class Filter {
+	const ITEM_IS_ENABLED = 'ln_collection_enable_filter_format';
```

Comment:
The constant name?

> On Mar 31, 2016, at 7:12 AM, Adam Fenton <notifications@github.com> wrote:
> 
> In src/Collection/Filter.php <https://github.com/moxie-leean/wp-endpoints-collection/pull/2#discussion_r58050346>:
> 
> > @@ -0,0 +1,12 @@
> > +<?php namespace Lean\Endpoints\Collection;
> > +
> > +/**
> > + * Custom filters used on this endpoint.
> > + *
> > + * @package Endpoints\Collection;
> > + * @since 0.1.0
> > + */
> > +class Filter {
> > +	const ITEM_IS_ENABLED = 'ln_collection_enable_filter_format';
> We should prefix with ln_endpoints_ to be consitent
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-endpoints-collection/pull/2/files/79212b057c9e0b6231ca448bc9f4e34930f2b65e#r58050346>